frontend/src/components/NavBar.css

```diff
@@ -7,7 +7,7 @@
   display: flex;
   justify-content: flex-end;
   flex-direction: column;
-  position: absolute;
+  position: fixed;

```

Comment:
try sticky instead

Reply:
tried sticky, doesn't work. Navbar stays in the middle of the page.